internal/trace/trace.go

```diff
@@ -16,35 +16,95 @@ package trace
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"os"
+	"sort"
+	"strings"
 
 	"go.opencensus.io/trace"
-	"golang.org/x/xerrors"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/codes"
+	ottrace "go.opentelemetry.io/otel/trace"
 	"google.golang.org/api/googleapi"
 	"google.golang.org/genproto/googleapis/rpc/code"
 	"google.golang.org/grpc/status"
 )
 
-// StartSpan adds a span to the trace with the given name.
+const (
+	telemetryPlatformTracingOpenCensus    = "opencensus"
+	telemetryPlatformTracingOpenTelemetry = "opentelemetry"
+	telemetryPlatformTracingVar           = "GOOGLE_API_GO_EXPERIMENTAL_TELEMETRY_PLATFORM_TRACING"
+)
+
+var (
+	// TODO(chrisdsmith): What is the correct name for the OpenTelemetry tracer?
+	openTelemetryTracerName     string = "cloud.google.com/go/internal/trace"
```

Comment:
maybe just drop the internal/trace part?

Reply:
Done, thanks!